### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by brandonb927**
This PR updates all the references to the old battlesnakeio org, repo, and various social media/contact info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,75 @@
...
+4. View server logs with the `heroku logs` command:
+
+    ```shell
+    heroku logs --tail
+    ```
 
 ## Questions?
 
-Email [hello@battlesnake.com](mailto:hello@battlesnake.com), or tweet [@battlesnakeio](http://twitter.com/battlesnakeio).
+Email [hello@battlesnake.com](mailto:hello@battlesnake.com), or tweet [@battlesnakeio](http://twitter.com/BattlesnakeOfficial).
```
Comment (bvanvugt) [inline README.md]: Still using the old twitter handle.

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "BattleSnake Python",
   "description": "A simple BattleSnake AI written in Python.",
-  "repository": "https://github.com/sendwithus/battlesnake-python",
-  "website": "http://battlesnake.io",
-  "keywords": ["battlesnake","sendwithus"]
+  "repository": "https://github.com/BattlesnakeOfficial/starter-snake-python",
+  "website": "http://play.battlesnake.com",
```
Comment (bvanvugt) [inline app.json]: Let's bump to https too?

File: README.md [hunk 3]
```diff
@@ -1,66 +1,75 @@
...
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
-#### You will need...
+## Prerequisites
 
-* a working Python 2.7 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
-* experience [deploying Python apps to Heroku](https://devcenter.heroku.com/articles/getting-started-with-python#introduction)
-* [pip](https://pip.pypa.io/en/latest/installing.html) to install Python dependencies
+- a working Python 3.8 development environment ([getting started guide](http://hackercodex.com/guide/python-development-environment-on-mac-osx/))
```
Comment (bvanvugt) [inline README.md]: repo is not actually 3.8 ready yet, keep at 2.7 for now

Comment (brandonb927) [reply README.md]: Oops, I cherry-picked this from a branch I was working on migrating to 3.8 😬

File: README.md (hunk 1 above)
Comment (brandonb927) [reply README.md]: I literally _just_ realized this because of an entirely different reason. Will fix!

**Comment by brandonb927** [comment]
@bvanvugt flagged issues should be resolved now in c6e66aa and 92ddcfc

**Pull request merged**